debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -561,7 +561,13 @@ public static SchemaRefreshMode parse(String value) {
             .withWidth(Width.MEDIUM)
             .withImportance(Importance.MEDIUM)
             .withDescription("The name of the Postgres logical decoding plugin installed on the server. " +
-                    "Supported values are '" + LogicalDecoder.DECODERBUFS.getValue() + "' and '" + LogicalDecoder.WAL2JSON.getValue() + "'. " +
+                    "Supported values are '" + LogicalDecoder.DECODERBUFS.getValue()
+                    + "', '" + LogicalDecoder.WAL2JSON.getValue()
+                    + "', '" + LogicalDecoder.PGOUTPUT.getValue()
+                    + "', '" + LogicalDecoder.WAL2JSON_STREAMING.getValue()
+                    + "', '" + LogicalDecoder.WAL2JSON_RDS.getValue()
+                    + "' and '" + LogicalDecoder.WAL2JSON_RDS_STREAMING.getValue()
+                    + "'. " +
```

Comment:
Unrelated cleanup in separate commit. Is it okay to show all possible values?

Reply:
This is fine. I wonder if you could create a utility method that would format the allow values for any enum.

Reply:
I can, but after holidays.